install_config/routing_from_edge_lb.adoc

```diff
@@ -198,8 +198,10 @@ gateway):
 ----
 ====
 
-. Mark the ramp node as an unschedulable node so that no pods end up on the ramp
-node itself:
+. Optionally, if you do not plan on configuring the ramp node to be
+highly-available, mark the ramp node as unschedulable. Skip this step if you do
```

Comment:
You can drop the hyphens in highly-available (both instances) per the [IBM Style Guide](http://www.ibm.com/developerworks/library/styleguidelines/).